src/helpers/flare.ts

```diff
@@ -61,7 +61,16 @@ export const sendToDatadog = async (
       const errResponse: string = (err.response?.data.error as string) ?? ''
       const errorMessage = err.message ?? ''
 
-      throw Error(`Failed to send flare file to Datadog Support: ${errorMessage}. ${errResponse}\n`)
+      let message = `Failed to send flare file to Datadog Support: ${errorMessage}. ${errResponse}\n`
+      const code = err.response?.status
+      // The error message doesn't say why there was an error, so it's important to tell the user why the request failed.
```

Comment:
What do you mean by this?

Reply:
Good question. Before this PR, the message just says:
`[Error] Failed to send flare file to Datadog Support: Request failed with status code 500.`
There is not any explanation as to why the status code is 500 though.
I will make sure to update the comment to clarify this